conda_smithy/feedstock_content/.gitignore

```
@@ -1,3 +1,13 @@
-*.pyc
+# User content belongs under recipe/.
+# Feedstock configuration goes in `conda-forge.yml`
+# Everything else is managed by the conda-smithy rerender process.
+# Please do not modify
+
+*
+!conda-forge.yml
 
-build_artifacts
+!*/
+!recipe/**
+!.ci_support/**
```

We could make this uniform with the suggested `!/conda-forge.yml` above:
```suggestion
!/*/
!/recipe/**
!/.ci_support/**
```
. Though it would technically not be needed since, AFAIK, anything with a `/` in it is in any case relative to `.gitignore`.
But no strong opinion from my side -- feel free to apply or ignore the suggestion.